scripts/migrate-v1-v2.js

```diff
@@ -63,12 +63,18 @@ for(const board of oldConfig) {
                     s.accountName = twitterName;
                     return s;
                 case 'tweeting': {
-                    return {
+                    const sourceDefinition = {
                         columns: s.columns,
                         type: 'publish',
                         accountType: 'twitter',
                         accountName: twitterName
                     };
+
+                    if (s.schedule && s.schedule.length > 0) {
```

Comment:
This repo isn't in mozilla style so far, it's in freaktechnik style ;) so no space after if.